include/oneapi/dpl/pstl/omp/util.h

```diff
@@ -83,7 +83,7 @@ class __buffer
 constexpr std::size_t __default_chunk_size = 2048;
 
 // Convenience function to determine when we should run serial.
-template <typename _Iterator, std::enable_if_t<!std::is_integral<_Iterator>::value, bool> = true>
+template <typename _Iterator, std::enable_if_t<!std::is_integral_v<_Iterator>, bool> = true>
```

Comment:
`std` -> `::std`

Reply:
Agree, but not in this PR: I have separate branch for this purpose.